### PR TITLE
Add performance test and make this work on Mac OS X.

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

File: src/test/runTest.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 pushd `dirname $0`
 export LD_LIBRARY_PATH=/opt/circonus/lib/amd64:/opt/circonus/lib
+export DYLD_LIBRARY_PATH=/opt/circonus/lib/amd64:/opt/circonus/lib:..
```
Comment (aidancully) [inline src/test/runTest.sh]: should LD_LIBRARY_PATH and DYLD_LIBRARY_PATH be the same?

Comment (postwait) [inline src/test/runTest.sh]: I don't know.  Perhaps not. If it worked before, then it isn't needed.

Comment (aidancully) [inline src/test/runTest.sh]: It worked before only when libcircllhist.so was installed when this script is run. I guess it doesn't interfere, but  MacOS and other Unixes should probably behave the same.

**Comment by aidancully** [comment]
lgtm

**Pull request merged**